# Glade API 67-to-68 upgrade delta for MIT-licensed @lwc/shared 9.4.3.
# It applies only to trees already at the pinned API 67 identity postimages.
# Salesforce copyright and MIT headers in the patched package files are retained.
# This adds version identity only; it does not claim full API 68 engine parity.
--- a/node_modules/@lwc/shared/dist/index.js
+++ b/node_modules/@lwc/shared/dist/index.js
@@ -276,2 +276,3 @@
     67 /* APIVersion.V67_262_SUMMER_26 */,
+    68 /* APIVersion.V68_264_WINTER_27 */,
 ];
--- a/node_modules/@lwc/shared/dist/index.cjs
+++ b/node_modules/@lwc/shared/dist/index.cjs
@@ -280,2 +280,3 @@
     67 /* APIVersion.V67_262_SUMMER_26 */,
+    68 /* APIVersion.V68_264_WINTER_27 */,
 ];
--- a/node_modules/@lwc/shared/dist/api-version.d.ts
+++ b/node_modules/@lwc/shared/dist/api-version.d.ts
@@ -11,2 +11,3 @@
-    V67_262_SUMMER_26 = 67
+    V67_262_SUMMER_26 = 67,
+    V68_264_WINTER_27 = 68
 }
